pdp/policy.go

```diff
@@ -63,12 +69,15 @@ func (p *Policy) describe() string {
 	ruleIdx := 0
```

Comment:
Policy set also has describe method. Why didn't change it as well?

Reply:
Line above allocates array of the same length as array of rules has. It can be to big array of for example million items. You should allocate no more than nDisplayRuleLimit + 1.